go.mod

```diff
@@ -1,4 +1,4 @@
-module pinctrl
+module github.com/viam-modules/pinctrl
```

Comment:
is changing the module name intentional?

Reply:
Yes. If you want your code to be importable by someone else, you need a universally unique name here. They're usually named after the github repo that should be considered the canonical version of the code.